multiqc/modules/picard/MarkDuplicates.py

```diff
@@ -90,7 +90,7 @@ def save_table_results(s_name, base_s_name, keys, parsed_data, recompute_merged_
             #
             if "markduplicates" in l.lower() and "input" in l.lower():
                 # Pull sample name from input
-                fn_search = re.search(r"INPUT(?:=|\s+)(\[?[^\s]+\]?)", l, flags=re.IGNORECASE)
+                fn_search = re.search(r"INPUT(?:=|\s+)(\[?[^\s]+]?)", l, flags=re.IGNORECASE)
```

Comment:
Why did you remove the escape character?

```suggestion
                fn_search = re.search(r"INPUT(?:=|\s+)(\[?[^\s]+\]?)", l, flags=re.IGNORECASE)
```

Reply:
Committing so that I can merge. Shout if you put this here intentionally.